modules/auth_email/utils/code_management.ts

```diff
@@ -0,0 +1,99 @@
+import { RuntimeError, ScriptContext } from "../module.gen.ts";
+
+function generateCode(): string {
```

Comment:
this should use crypto to generate the code. (ik i wrote this code)

Reply:
Yep, uses `Module.tokens.generateRandomCodeSecure` now!